bin/prepare_windows_host_for_app_distribution.ps1

```diff
@@ -27,6 +27,14 @@ param (
 # Stop script execution when a non-terminating error occurs
 $ErrorActionPreference = "Stop"
 
+$amiVersion = [Environment]::GetEnvironmentVariable('AMI_VERSION', 'Machine')
+Write-Output "AMI_VERSION is: $amiVersion"
+if (![string]::IsNullOrWhiteSpace($amiVersion) -and [version]$amiVersion -ge [version]'0.2') {
+  Write-Output "Tooling is already pre-installed in AMI versions >= 0.2. Only installing code signing certificate."
+  & "setup_windows_code_signing.ps1"
+  Exit 0
```

Comment:
I wonder if `Exit $LastExitCode` would be more appropriate here. 

I remember looking into the exit code management before sabbatical, but I don't remember what I learned or if/where I tracked my findings.

Logged https://linear.app/a8c/issue/AINFRA-1466/audit-ci-toolkit-ps1-scripts-for-appropriate-exit-code-reporting to followup.